app/bingey-api/controllers/title.js

```diff
@@ -2,6 +2,17 @@ const { check, oneOf, validationResult } = require('express-validator');
 
 const Title = require('$/models/title');
 
+
+const getTitleById = async (req, res) => {
+  try {
+    const title = await Title.findById(req.params.id);
+
+    res.json(title);
+  } catch (err) {
+    res.json({ errors: errors });
```

Comment:
This throws a `ReferenceError: errors is not defined`.
It should be 
```suggestion
    res.json({ errors: err });
```